bin/reth/src/args/payload_builder_args.rs

```diff
@@ -46,6 +46,20 @@ pub struct PayloadBuilderArgs {
     pub compute_pending_block: bool,
 }
 
+impl Default for PayloadBuilderArgs {
+    fn default() -> Self {
+        Self {
+            extradata: default_extradata(),
+            max_gas_limit: 30_000_000,
+            interval: Duration::from_secs(1),
+            deadline: Duration::from_secs(12),
+            max_payload_tasks: 3,
+            #[cfg(feature = "optimism")]
+            compute_pending_block: false,
```

Comment:
we should use the config's default here

https://github.com/paradigmxyz/reth/blob/5ac4a3d4cb33fd2f6c9219f0da83088a4186da59/crates/payload/basic/src/lib.rs#L306-L321